.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -13,15 +13,18 @@ Reviewing guidelines: https://hydra.nixos.org/job/nixpkgs/trunk/manual/latest/do
 
 ###### Things done
 
-<!-- Please check what applies. Note that these are not hard requirements but merely serve as information for reviewers. -->
+<!-- 
+Please check what applies. Note that these are not hard requirements but merely serve as information for reviewers. 
+You might find this discussion useful: https://discourse.nixos.org/t/making-the-pull-request-template-more-beginner-friendly-nixpkgs/388
```

Comment:
We should point to documentation in this repository rather than discourse discussions.
There is .github/CONTRIBUTING.md if you want to add something.